### PR TITLE
Correct metadata for plate 3 and rerun processing features

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
After speaking to Cameron, I noticed that the data on his end showed values as None for the `Metadata_genotype`. This prompted an investigation into the metadata and why cells had `None` type metadata. 

I noticed that the wells in the platemap contained `0`, which I believe I added prior to analysis due to prior experience with another project needing to have `0` in the well string to work. This was an easy correction. 

What was strange was that in the platemap for plates 3 and 3 prime, the string `NULL` (in all caps) was becoming `nan` when loaded in as a pandas dataframe. But, in the platemaps for plates 1 and 2, I had created them using `Null` (not all caps) and did not have this problem. Even after looking into this with Cameron, we found nothing about pandas reading string `NULL` (all caps) as None types.

I have now fixed the metadata for the platemap for plates 3 and 3 prime and it has been uploaded to figshare (along with fixing the version number in the download data module).

### CODE REVIEW

File: 2.cellprofiler_analysis/README.md [hunk 1]
```diff
@@ -19,3 +19,15 @@ source nf1_analysis.sh
 To run analysis on plates 1 and 2, it took approximately one hour. 
 To run analysis on plates 3 and 3 prime, it took approximately 31 hours.
 The analysis was run on a Linux-based machine running Pop_OS! LTS 22.04 with an AMD Ryzen 7 3700X 8-Core Processor.
+
+## Accessing the CellProfiler output - SQLite files
+
+We used Git LFS to store the large files like SQLite files.
+If you would like to access these files after cloning the repo, you will need to run the below command in terminal. 
+
+**Note:** Make sure that you have Git LFS installed on your local machine. Follow the instructions on [the GitHub Docs](https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage).
```
Comment (MattsonCam) [inline 2.cellprofiler_analysis/README.md:28]: Good job documenting this.

**Review by MattsonCam — APPROVED** [review]
Overall, great documentation coverage! Your code is clear, and your comments communicate what you are trying to do.

File: 2.cellprofiler_analysis/README.md (hunk 1 above)
Comment (jenna-tomkinson) [reply 2.cellprofiler_analysis/README.md:28]: Thank you!

**Comment by jenna-tomkinson** [comment]
@MattsonCam Thanks for the review! I am glad this fixed the issue with the metadata on plates 3 and 3 prime.

I will be merging now!

**Pull request merged**